File: src/main/java/ru/sicampus/bootcamp2026/dto/response/InvitationResponse.java
package ru.sicampus.bootcamp2026.dto.response;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class InvitationResponse {
    private Long id;
    private MeetResponse meet;
    private UserResponse user;
    private String status; // "PENDING", "ACCEPTED", "DECLINED"
    private LocalDateTime respondedAt;
}

File: server/src/main/kotlin/com/nikshet/industrialbackend/dto/response/InvitationResponse.kt
package com.nikshet.industrialbackend.dto.response

import com.nikshet.industrialbackend.entity.enums.InvitationStatus
import java.time.Instant
import java.util.*

data class InvitationResponse(
    val id: UUID,
    val meetingId: UUID,
    val meetingTitle: String,
    val meetingStartTime: Instant,
    val meetingLocation: String,
    val status: InvitationStatus,
    val invitedAt: Instant,
    val respondedAt: Instant?
)